bom/runtime/pom.xml

```diff
@@ -68,7 +68,7 @@
            is not available in Maven fast enough: -->
         <graal-sdk.version>19.1.1</graal-sdk.version>
         <gizmo.version>1.0.0.Alpha5</gizmo.version>
-        <jackson.version>2.9.9</jackson.version>
+        <jackson.version>2.9.9.20190727</jackson.version>
```

Comment:
Are you sure about this one? I don't see it on Central but I see a `2.9.9.2`.

https://search.maven.org/search?q=g:com.fasterxml.jackson.core%20AND%20a:jackson-databind&core=gav

Reply:
http://central.maven.org/maven2/com/fasterxml/jackson/jackson-bom/2.9.9.20190727/

Reply:
mmh... let me fix this properly, pinging you in chat